[workflow] Update issue templates (#1794)
taichi-dev/taichi: .github/ISSUE_TEMPLATE/bug_report.md

A clear and concise description of what the bug is, ideally within 20 words.

**To Reproduce**
Please post a **minimal sample code** to reproduce the bug.
The developer team will put a higher priority on bugs that can be reproduced within 20 lines of code. If you want a prompt reply, please keep the sample code **short** and **representative**.

```py
# sample code here
```

**Log/Screenshots**
Please post the **full log** of the program (instead of just a few lines around the error message, unless the log is > 1000 lines). This will help us diagnose what's happening. For example:

```
$ python my_sample_code.py
[Taichi] mode=release
[Taichi] version 0.6.29, llvm 10.0.0, commit b63f6663, linux, python 3.8.3
...
```

**Additional comments**

> If possible, please also consider attach the output of command `ti diagnose` for obtaining detailed environment informations to help us diagnose.

Reviewer: attaching
```suggestion
If possible, please also consider attaching the output of command `ti diagnose`. This produces the detailed environment information and helps us diagnose faster.
```
